Framework/DataObjects/test/TimeSplitterTest.h

```diff
@@ -180,4 +183,50 @@ class TimeSplitterTest : public CxxTest::TestSuite {
     TS_ASSERT(!roi.empty());
     TS_ASSERT_EQUALS(roi.numBoundaries(), 4);
   }
+
+  void testTimeSplitterFromMatrixWorkspace() {
```

Comment:
Could you name it `test_TimeSplitterFromMatrixWorkspace` similar to the other tests in this file.